src/angular-materialize.js

```diff
@@ -379,6 +379,9 @@
                                 }
                             });
                         }
+                        
+                        if(attrs.ngDisabled)
+                                scope.$watch(attrs.ngDisabled, initSelect)
```

Comment:
Please add curly brackets around the statement, this is how it is done in the rest of the file, and it is clearer that way.